components/docbook/sgml-common/Makefile

```diff
@@ -39,12 +38,8 @@ COMPONENT_PREP_ACTION +=(cd $(@D) && \
 # Missing files in build dir for configure without this.
 COMPONENT_PRE_CONFIGURE_ACTION =        ($(CLONEY) $(SOURCE_DIR) $(@D))
 
```

Comment:
It looks like --disable-dependency-tracking is needed.

Reply:
Hmm, not on oibuild:
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating bin/Makefile
config.status: creating bin/install-catalog
config.status: creating config/Makefile
config.status: creating doc/Makefile
config.status: creating doc/HTML/Makefile
config.status: creating doc/man/Makefile
config.status: creating doc/refentry/Makefile
config.status: creating sgml-iso-entities/Makefile
config.status: creating xml-iso-entities/Makefile
/usr/bin/touch /ws/klausz/oi-userland/components/docbook/sgml-common/build/amd64/.configured

Reply:
But on oi.buildzone with Jenkins.

Reply:
Okay added wanted option to Makefile